dhctl/pkg/kubernetes/actions/manifests/manifests.go

```diff
@@ -69,189 +73,196 @@ func GetDeckhouseDeployTime(deployment *appsv1.Deployment) time.Time {
 }
 
 func ParametrizeDeckhouseDeployment(deployment *appsv1.Deployment, params DeckhouseDeploymentParams) *appsv1.Deployment {
-	deployment.Spec.Template.Spec.Containers[0].Image = params.Registry
+	deckhousePodTemplate := deployment.Spec.Template
+	deckhouseContainer := deployment.Spec.Template.Spec.Containers[0]
+	deckhouseContainerEnv := deckhouseContainer.Env
+
+	freshDeployment := params.DeployTime.IsZero()
 
-	deployTime := params.DeployTime
-	if deployTime.IsZero() {
-		deployTime = time.Now()
+	if freshDeployment {
+		params.DeployTime = time.Now()
 	}
 
-	for i, env := range deployment.Spec.Template.Spec.Containers[0].Env {
```

Comment:
We are using `ParametrizeDeckhouseDeployment` in two cases:

1. Parameterize deployment from `DeckhouseDeployment` function when we first run bootstrap and create deployment
2. When we've restarted bootstrap, we get deployment from cluster and parameterize received deployment. We are doing it for prevent Deckhouse restart after restart bootstrap. 

Therefore we are looking for env variables, volumes, etc, but not add they to arrays with append.

I think, we need to revert parameterize logic.

Reply:
I think, it is my bad, because i did not leave a comments to explain logic.
You can add it.

Reply:
Don't blame yourself. From my point of view, removing these checks didn't look like a problem. However, after reading your comment, I decided to add a test for mutation, and it turned out I had to fix my code. Thank you for being observant. Fixed.